src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -29,7 +30,12 @@
                         KBase environment (default: PROD)
   --upload_file_ext UPLOAD_FILE_EXT [UPLOAD_FILE_EXT ...]
                         Upload only files that match given extensions (default: ['genomic.fna.gz'])
-  --workers WORKERS     Number of workers for multiprocessing (default: 5)
+  --batch_size BATCH_SIZE
+                        Number of files to upload per batch (default: 2500)
+  --jr_max_tasks JR_MAX_TASKS
+                        The maxmium subtasks for the callback server (default: 20)
```

Comment:
I would call this cbs_max_tasks or change the docs to say "callback server / job runner". A new user isn't going to know what `jr` means.

Reply:
Updated!

Reply:
:+1: